tracer/src/Datadog.Trace.Tools.Runner/CheckIisCommand.cs

```diff
@@ -0,0 +1,147 @@
+// <copyright file="CheckIisCommand.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Tools.Runner.Checks;
+using Microsoft.Web.Administration;
+using Spectre.Console;
+using Spectre.Console.Cli;
+
+using static Datadog.Trace.Tools.Runner.Checks.Resources;
+
+namespace Datadog.Trace.Tools.Runner
+{
+    internal class CheckIisCommand : AsyncCommand<CheckIisSettings>
+    {
+        public override Task<int> ExecuteAsync(CommandContext context, CheckIisSettings settings)
+        {
+            return ExecuteAsync(settings, null, null);
+        }
+
+        public override ValidationResult Validate(CommandContext context, CheckIisSettings settings)
+        {
+            var result = base.Validate(context, settings);
+
+            if (result.Successful)
+            {
+                // Perform additional validation
+                if (settings.SiteName.Count(c => c == '/') > 1)
+                {
+                    return ValidationResult.Error($"IIS site names can't have multiple / in their name: {settings.SiteName}");
+                }
+            }
+
+            return result;
+        }
+
+        internal static async Task<int> ExecuteAsync(CheckIisSettings settings, string applicationHostConfigurationPath, int? pid)
+        {
+            var values = settings.SiteName.Split('/');
```

Comment:
I don't think it should be an issue, but maybe string splitoptions to be sure?
```suggestion
            var values = settings.SiteName.Split('/', StringSplitOptions.RemoveEmptyEntries);
```

Reply:
I can't think of a scenario where that could be useful. There can be one slash at most (this is validated by the command) and `site/` is a valid input properly handled by the code